apps/condo/domains/organization/integrations/sbbol/sync/features/index.js

```diff
@@ -0,0 +1,57 @@
+const concat = require('lodash/concat')
+const get = require('lodash/get')
+const uniq = require('lodash/uniq')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const { SERVICE_PROVIDER_PROFILE_FEATURE, ALL_FEATURES } = require('@condo/domains/organization/constants/features')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const { syncServiceProviderProfileState } = require('./serviceProviderProfile')
+
+const logger = getLogger('sbbol/syncFeatures')
+
+
+/**
+ * Updates organization with new features and creates all needed contexts if needed
+ * @param context keystone context acquired from main sync task containing adminContext for server-side utils
+ * @param organization created / updated organization
+ * @param {Array<string>} features list of organization features
+ * @returns {Promise<void>}
+ */
+async function syncFeatures ({ context, organization, features }) {
+    const { context: adminContext } = context
+    const orgId = organization.id
+    // Get current feature-list
+    const existingFeatures = get(organization, 'features', [])
+    logger.info({ msg: `Staring feature-sync process for organization: ${orgId}`, orgId, existingFeatures })
+    // Remove out-dated features from existing feature list
+    const actualExistingFeatures = existingFeatures.filter(feat => ALL_FEATURES.includes(feat))
+    // Remove out-dated features from incoming feature list
+    const actualFeaturesToConnect = features.filter(feat => ALL_FEATURES.includes(feat))
+    const connectedFeatures = []
+
+    // Feature-specific logic
+    if (actualFeaturesToConnect.includes(SERVICE_PROVIDER_PROFILE_FEATURE)) {
+        const success = await syncServiceProviderProfileState({ context, organization })
+        if (success) {
+            logger.info({ msg: `${SERVICE_PROVIDER_PROFILE_FEATURE} feature is successfully connected` })
+            connectedFeatures.push(SERVICE_PROVIDER_PROFILE_FEATURE)
+        }
+    }
+
+    const actualFeatures = uniq(concat(actualExistingFeatures, connectedFeatures))
```

Comment:
why lodash?
    const actualFeatures = [...new Set(actualExistingFeatures.concat(connectedFeatures))]

Reply:
> Lodash is good, so more lodash = more good 😄 

We have no convention here, and I'm sure that if I had used the 'new Set' option, someone like @akarjakin  would have come along and said "Why not lodash?"))0)

For me in this specific case this was more clear to read